utils/cliutils/commandsflags.go

```diff
@@ -857,6 +858,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  transitive,
 		Usage: "[Default: false] Set to true to look for artifacts also in remote repositories. The search will run on the first five remote repositories within the virtual repository. Available on Artifactory version 7.17.0 or higher.` `",
 	},
+	searchInclude: cli.StringFlag{
+		Name:  searchInclude,
+		Usage: fmt.Sprintf("[Optional] List of fields in the form of \"value1;value2;...\". Only the path and the fields that are specified will be returned. The fields must be part of the 'items' AQL domain. for the full supported items list  see %s jfrog-artifactory-documentation/artifactory-query-language` `", coreutils.JFrogHelpUrl),
```

Comment:
* ... for the full... --> ... For the full...
* ... supported items list  see... --> ... supported items list, see...
* Should there be a white-space afer `%s`?